Python/samples/house-energy/main.py

```diff
@@ -371,7 +383,7 @@ def main(render: bool = False, config_setup: bool = False):
 
     if args.test_local:
         test_random_policy(
```

Comment:
did you notice how the initial conditions are not logged? Do you think that's maybe important?
```
sim_state = sim.episode_start(config=scenario_configs[episode-1])
        sim_state = sim.get_state()
        if log_iterations:
            action = policy(sim_state)
            for key, value in action.items():
                action[key] = None
            sim.log_iterations(sim_state, action, episode, iteration)
        print(f"Running iteration #{iteration} for episode #{episode}")
        iteration += 1
        while not terminal:
            action = policy(sim_state)
            sim.episode_step(action)
            sim_state = sim.get_state()
            if log_iterations:
                sim.log_iterations(sim_state, action, episode, iteration)

Reply:
> did you notice how the initial conditions are not logged? Do you think that's maybe important?
> 
> ```
> sim_state = sim.episode_start(config=scenario_configs[episode-1])
>         sim_state = sim.get_state()
>         if log_iterations:
>             action = policy(sim_state)
>             for key, value in action.items():
>                 action[key] = None
>             sim.log_iterations(sim_state, action, episode, iteration)
>         print(f"Running iteration #{iteration} for episode #{episode}")
>         iteration += 1
>         while not terminal:
>             action = policy(sim_state)
>             sim.episode_step(action)
>             sim_state = sim.get_state()
>             if log_iterations:
>                 sim.log_iterations(sim_state, action, episode, iteration)
> ```

Good catch. `log_iterations` logs `self.sim_config`, which is set inside `episode_start`. This could be cleaner. Let me look at how it's done in cartpole.